### PR TITLE
DES-11128 - resize notification transparent window manager & hide it …

### PR DESCRIPTION
**Pull request by rbarros-daitan**
…when not needed

### CODE REVIEW

File: minuet/Paragon.Plugins.Notifications/Paragon.Plugins.Notifications.csproj [hunk 1]
```diff
@@ -1,179 +1,177 @@
...
-      <Name>Paragon.Plugins.MessageBus</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Paragon.Runtime\Paragon.Runtime.csproj">
-      <Project>{766A5435-5221-40FF-825E-BE001A7D6662}</Project>
-      <Name>Paragon.Runtime</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup />
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (lneir) [inline minuet/Paragon.Plugins.Notifications/Paragon.Plugins.Notifications.csproj:1]: was anything changed or just spacing changes?

File: minuet/Paragon.Plugins.Notifications/Themes/SymphonyNotification.xaml [hunk 2]
```diff
@@ -114,7 +114,7 @@
             <MultiDataTrigger>
                 <MultiDataTrigger.Conditions>
                     <Condition Binding="{Binding Path=IsRemoving}" Value="True" />
-                    <Condition Binding="{Binding Path=Position, Converter={StaticResource PositionLeftOrRightConverter}}" Value="Right" />
+                    <Condition Binding="{Binding Path=Position, Converter={StaticResource PositionLeftOrRightConverter}}" Value="Left" />
```
Comment (lneir) [inline minuet/Paragon.Plugins.Notifications/Themes/SymphonyNotification.xaml:117]: why left now?

File: minuet/Paragon.Plugins.Notifications/Views/NotificationWindow.xaml.cs [hunk 3]
```diff
@@ -33,12 +33,40 @@ public NotificationWindow()
...
-            Win32Api.AddToolWindowStyle(this.handle);
-        }
+        }
 
-        public void ShowOnMonitor(IMonitor monitor)
+		//DES-11128
+        public void ShowOnMonitor(RequestShowEventArgs args)
         {
-            Win32Api.MoveAndShowWithNoActivate(handle, monitor);
+            //Win32Api.MoveAndShowWithNoActivate(handle, args);
```
Comment (lneir) [inline minuet/Paragon.Plugins.Notifications/Views/NotificationWindow.xaml.cs]: nice to get rid of native method calls.

File: minuet/Paragon.Plugins.Notifications/Views/NotificationWindow.xaml.cs [hunk 4]
```diff
@@ -33,12 +33,40 @@ public NotificationWindow()
...
+            this.Height = 56 * args.NotificationCount;
+            this.Width = 300;
```
Comment (lneir) [inline minuet/Paragon.Plugins.Notifications/Views/NotificationWindow.xaml.cs]: anyway we can parameterize this so we aren't hard coding?  or use const?

File: minuet/Paragon.Plugins.Notifications/Themes/SymphonyNotification.xaml (hunk 2 above)
Comment (rbarros-daitan) [reply minuet/Paragon.Plugins.Notifications/Themes/SymphonyNotification.xaml:117]: This panel was a two columns. with resizing, it was consolidated into a single panel and based on left/right notification, it changes alignment.

File: minuet/Paragon.Plugins.Notifications/Views/NotificationWindow.xaml.cs (hunk 3 above)
Comment (rbarros-daitan) [reply minuet/Paragon.Plugins.Notifications/Views/NotificationWindow.xaml.cs]: ok. done.

File: minuet/Paragon.Plugins.Notifications/Views/NotificationWindow.xaml.cs (hunk 4 above)
Comment (rbarros-daitan) [reply minuet/Paragon.Plugins.Notifications/Views/NotificationWindow.xaml.cs]: Size is defined on xaml. The resize on height takes its primary value and multiplies by number of notification. Is approach this ok?

File: minuet/Paragon.Plugins.Notifications/Paragon.Plugins.Notifications.csproj (hunk 1 above)
Comment (rbarros-daitan) [reply minuet/Paragon.Plugins.Notifications/Paragon.Plugins.Notifications.csproj:1]: The removal of native code C# files might have forced some change on this file.

**Comment by lneir** [comment]
lgtm

**Pull request merged**